assets/helpers/subscriptions.js

```diff
@@ -125,6 +125,54 @@ const paperSubscriptionPrices = {
   deliverySunday: GBP(15.12),
 };
 
+export type WeeklyPromoCode = '10ANNUAL';
```

Comment:
This doesn't look like it should be a new type to me, there is already a `PromoCodes` type in externalLinks.js, surely this is an instance of that?

Reply:
This was intended to guard against any other value than `10ANNUAL` being used. I've just made that a string type now because where it's used it's always a string, and switching to a `PromoCodes` type requires making deeper changes, which may not be worth it yet as I'm not sure how long this promo will be the default for.

This does mean there's a weakness at https://github.com/guardian/support-frontend/pull/1330/files#diff-7bdd389e4027e883c04ab0d735f2d779R259 because `promoCode` could be anything and may not exist in `subscriptionPromoPricesForGuardianWeekly`.